server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -227,6 +236,8 @@ public DeviceStats(StreamInput in) throws IOException {
             previousSectorsRead = in.readLong();
             currentSectorsWritten = in.readLong();
             previousSectorsWritten = in.readLong();
+            currentIOTime = in.readLong();
+            previousIOTime = in.readLong();
```

Comment:
Version checks for bwc are needed here as well

Reply:
Thanks for review,  Version checks is added